x11/xcowsay/Portfile

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+
+github.setup            nickg xcowsay 1.4 v
+github.tarball_from     releases
+name                    xcowsay
```

Comment:
Sorry for missing this earlier. This `name` line is redundant and should be removed. It is already provided by the GitHub portgroup.